### PR TITLE
Dom

### PR DESCRIPTION
**Pull request by YarHadi**
### DOM

[Demo](https://yarhadi.github.io/dom/) |
[Code base](https://github.com/YarHadi/dom/tree/v2)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work. Add some functionality to your code. Create DOM elements. Use event listener. Work with dataset property of elements.

**Comment by YarHadi** [comment]
Hello, Master. Your comments was resolved.
Please, check again.

File: submissions/YarHadi/task_js_dom/form.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+createButtons(characters);
+
+// Creation of content
+
+const mainContainer = document.querySelector(".main-content");
+
+function showContent(buttonId, data) {
+  const contentData = data.find((character) => character.id == buttonId);
+  const content = `<h1 class="text-header" id="main-header">${contentData.header}</h1>
+  <img
```
Comment (madmaxWMFU) [inline submissions/YarHadi/task_js_dom/form.js:64]: Also, read about [desrtuction](https://javascript.info/destructuring-assignment)

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

File: submissions/YarHadi/task_js_dom/form.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const onClick = ({ target }) => {
+  const selectedBtn = target.dataset.id;
+  showContent(selectedBtn, characters);
+};
+
+navMenu.addEventListener("click", onClick);
+
+//inner first page
+
+showContent(1, characters);
```
[the remark this answers is not in the document]
Comment (YarHadi) [reply submissions/YarHadi/task_js_dom/form.js]: I did not understand a little, but I did something)) 
Please check.

**Comment by YarHadi** [comment]
Almost all comments was resolved)
Please check.

File: submissions/YarHadi/task_js_dom/form.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  showContent(header,img,text);
+};
+
+navMenu.addEventListener("click", onClick);
+
+//inner first page
+
+function firstPage(data){
+  const {header,img,text} = data.find((character) => character.id == 1);
+  showContent(header,img,text);
```
Comment (madmaxWMFU) [inline submissions/YarHadi/task_js_dom/form.js:90]:
```suggestion
function initPage(){
  const [{ header, img, text }] = character;
  showContent(header,img,text);
```

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

**Comment by YarHadi** [comment]
Good morning.
The work is done.
Please check again.

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Comment by YarHadi** [comment]
Thank you. Have a nice day.

**Pull request merged**